docs/source/algorithms/LoadElementalAnalysisData-v1.rst

```diff
@@ -0,0 +1,22 @@
+﻿.. algorithm::
+
+.. summary::
+
+.. relatedalgorithms::
+
+.. properties::
+
+Description
+-----------
+
+The LoadElementalAnalysisData algorithm loads data from .dat files for the run chosen into a single
+:ref:`WorkspaceGroup <WorkspaceGroup>` containing a :ref:`Workspace <Workspace>` for each Detector identified. Each :ref:`Workspace <Workspace>` contains the identified spectra 
+results. The name of the :ref:`WorkspaceGroup <WorkspaceGroup>` depends on the setting of GroupWorkspace.
+
+The algorithm searches the Data Search Directories for .dat files for the run specified. File names must be in the format 'ral[run number].rooth[detector number, spectra number].dat'.
+A run number and name for the GroupWorkspace must be provided.
+
```

Comment:
can we have an example python script?

Reply:
I've had a look into this. Other load examples (e.g. Load - https://docs.mantidproject.org/nightly/algorithms/Load-v1.html) require users to download data to recreate the examples. I've checked the current download and there are no suitable files for the LoadElementalAnalysisData algorithm. We'd need to explore adding files to the download first.

Reply:
Thats a shame, leave it then